x/ibc/types/keys.go

```diff
@@ -21,22 +21,18 @@ const (
 
 // KVStore key prefixes for IBC
 var (
-	KeyClientPrefix            = []byte("clientState")
-	KeyClientTypePrefix        = []byte("clientType")
-	KeyConsensusStatePrefix    = []byte("consensusState")
-	KeyClientConnectionsPrefix = []byte("clientConnections")
-	KeyConnectionPrefix        = []byte("connection")
+	KeyClientStorePrefix = []byte("clients")
+	KeyConnectionPrefix  = []byte("connections")
 )
 
 // KVStore key prefixes for IBC
 const (
-	KeyChannelPrefix int = iota + 1
-	KeyChannelCapabilityPrefix
-	KeyNextSeqSendPrefix
-	KeyNextSeqRecvPrefix
-	KeyPacketCommitmentPrefix
-	KeyPacketAckPrefix
-	KeyPortsPrefix
+	KeyChannelPrefix           = "channelEnds"
```

Comment:
replaced iota prefixes with string prefixes to ensure consistency across all implementations

Even though channels are the only thing that need a separate prefix for efficient iteration, explained here: https://github.com/cosmos/ics/issues/387#issuecomment-604791157

I kept prefixes for all `04/channel` state in order to create a uniform key format